pkgs/development/tools/conftest/default.nix

```diff
@@ -15,13 +15,25 @@ buildGoModule rec {
 
   doCheck = false;
 
-  buildFlagsArray = ''
-    -ldflags=
-        -X main.version=${version}
-  '';
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X github.com/open-policy-agent/conftest/internal/commands.version=${version}"
```

Comment:
Is it possible to document these? Using a long form would be ideal but if not just giving a couple of words each helps readers looking for something quickly.

Reply:
I'm not sure it's worth documenting `buildFlagsArray`/`ldflags` as most people packaging know what they are and they're easy to look up in `ld` documentation. As the variable name says they're flags for `ld`.
Also there are many many many packages with almost identical inputs that aren't documented.

I wouldn't add a comment above `vendorSha256` explaining what it's for or add a comment for why there's `rec` in `buildGoModule rec {`

I'm happy to remove `-s` and `-w` though as now that I think about it they're not particularly necessary, it was mostly leftover from another derivation

Reply:
Yeah. Removing `-s` and `-w` makes sense, and removes two of the things to question. I honestly can't find out what `-X` is. I agree, `-ldflags` is obvious enough.